src/lib/config.js

```diff
@@ -55,26 +58,55 @@ function parseKeysConfig () {
         secret: 'iMx6i3D3acJPc4aJlK0iT/pkJP3T+Dqte9wg6hXpXEv08CpNQSm1J5AI6n/' +
           'QVBObeuQWdQVpgRQTAJzLLJJA/Q==',
         public: '9PAqTUEptSeQCOp/0FQTm3rkFnUFaYEUEwCcyyySQP0='
+      },
+      notificationSign: {
+        secret: fs.readFileSync(path.join(__dirname, '../../test/data/signKeyRSAPrv.pem'), 'utf8'),
+        public: fs.readFileSync(path.join(__dirname, '../../test/data/signKeyRSAPub.pem'), 'utf8')
+      }
+    }
+  } else {
+    const privateKeyPath = Config.getEnv(envPrefix, 'SIGNING_PRIVATE_KEY')
+    const publicKeyPath = Config.getEnv(envPrefix, 'SIGNING_PUBLIC_KEY')
+    if (!privateKeyPath) {
+      if (process.env.NODE_ENV === 'production') {
+        throw new Error('No ' + envPrefix.toUpperCase() + '_SIGNING_PRIVATE_KEY provided')
+      }
+      const keys = keypair()
```

Comment:
@clark800 you mentioned generating keys on headless could take a very long time. I haven't looked into where `keypair` gets its entropy, but that could be the case here, too. Maybe we should have a followup task for this and ed25519 keys in dev